src/dsp.jl

```diff
@@ -410,3 +411,64 @@ function mfilter(r, s)
   n = length(r) - 1
   sfilt(f, padded(s, (0, n)))[n+1:end]
 end
+
+"""
+$(SIGNATURES)
+Inverse Short Time Fourier Transform (ISTFT). 
+
+The signal windowing must obey the constraint of "nonzero overlap add" (NOLA). 
+Implementation based on Hristo (2019, 2020) and `istft` in `scipy`. 
+
+H. Zhivomirov. On the Development of STFT-analysis and ISTFT-synthesis Routines 
+and their Practical Implementation. TEM Journal, ISSN: 2217-8309, DOI: 10.18421/TEM81-07, 
+Vol. 8, No. 1, pp. 56-64, Feb. 2019. 
+(http://www.temjournal.com/content/81/TEMJournalFebruary2019_56_64.pdf)
+
+Hristo Zhivomirov (2020). Inverse Short-Time Fourier Transform (ISTFT) with Matlab 
```

Comment:
What is the relevance of the MATLAB version here?

Reply:
It is the MATLAB implementation of the ISTFT algorithm in Hristo 2019. I have removed the reference Hristo 2020 from the docstring.